pkgs/by-name/im/imsprog/package.nix

```diff
@@ -0,0 +1,54 @@
+{ lib, stdenv, fetchFromGitHub, cmake, makeWrapper, pkg-config, bash, gnome, libusb1, qt5, wget }:
+
+stdenv.mkDerivation rec {
+  pname = "IMSProg";
```

Comment:
```suggestion
  pname = "imsprog";
```

Reply:
Is it the norm to ditch the self given capitalization?

Reply:
Not so much a norm, it's more of a convention (one of the bullet points further down): https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

Reply:
@h7x4 thank you for the link! @Aleksanaa thank you for the feedback, I adjusted accordingly.